#	Skill Set for the Biotechnology & Pharmaceutical Industry



##	Graduates from Biology, Biochemistry, Biotechnology, Medicinal Chemistry, Pharmacy, and/or Pharmaceutical Science


Skill set for graduates from (advanced) degree programs in biology, biochemistry, biotechnology, medicinal chemistry, or pharmacy and pharmaceutical science
+ "How to speed up cancer geneomics multi-omics analysis" Webinar from Code Ocean:
	- Spans:
		* pharmcology
		* metabolomics
		* proteomics
		* transcriptomics
		* epigenomics
		* genomics
+ molecular modeling for computational protein engineering
	- Rosetta
	- Molecular Operating Environment, MOE
	- Schrodinger
+ write standard operating procedures (SOP) and high-quality electronic laboratory notebooks (ELN)
+ Manage a suite of instruments including transmission electron microscopes, focus ion beam scanning electron microscopes, cryo light microscopes, high pressure freezers, etc.
+ skill set:
	- engineering lab-on-chips, LoCs
	- engineering digital microfluidics, DMFs
+ Discrimination of commensals and pathogens by gut neuronal circuits.
+ Good understanding of high-throughput single cell assays such as single cell transcriptomics, single-cell epigenomics, FACS, and CyTOF.
+ Experience with RNA-seq, ATAC-seq and/or ChIP-seq experiments and data at the single cell level.
+ skill set:
	- BLAH.
+ skill set:
	- BLAH.
+ skill set:
	- BLAH.
+ skill set:
	- BLAH.
+ skill set:
	- BLAH.
+ skill set:
	- BLAH.
+ skill set:
	- BLAH.
+ skill set:
	- BLAH.
+ skill set:
	- BLAH.
+ skill set:
	- BLAH.
+ skill set:
	- BLAH.
+ skill set:
	- BLAH.
+ skill set:
	- BLAH.
+ skill set:
	- BLAH.
+ skill set:
	- BLAH.
























## R&D roles in Bio Design Automation, BDA


+ skill set:
	- Design and architect the backend for our computer aided genetic circuit pipeline, including the genetic compiler and simulation engine
		* This is ***bio design automation***, BDA.
	- Develop our cloud infrastructure, data warehouse, build and deploy system, and machine learning platform. You will drive discussion and decisions on microservices vs monoliths, managed services vs deployed, etc.
	- Work on tooling to help improve the efficiencies of our synthetic biologists including data analysis platforms and robotic automation
	- solid at communicating architectures internally through clear design documents
	- embracing ambiguity and driving for impact
	- excited to partner closely with and learn from our synthetic biology team
	- interested in helping define the company product, direction and culture
	- You are a highly skilled technical lead in software engineer with years of experience in industry. You are passionate about joining an early stage startup where autonomy, passion to learn and excitement to engineer biology take precedence over process and ego. If you have a background in genetics or cellular biology, great! If not, you have strong experience with ETL systems, data storage and access, cloud infrastructures, build and deploy systems with a passion to learn the science.
+ skill set:
	- You will lead key customer software efforts starting from requirements gathering and architecture formulation through development, testing, and deployment cycles. You will build technically and aesthetically crisp software in a primarily reactive web programming environment. The stack will vary based on customer specifications, so flexibility is preferred to strict adherence to one specific stack, but ReactJS + GraphQL + NoSQL + Typescript is our most utilized set of technologies. Coding quality and robust functionality is most important, regardless of language.
	- You will work with a team of experts in algorithms, software design, synthetic biology, and interface design. Your role will include the translation of algorithms and application requirements into feasible architectures, and ultimately, into code that is performant, readable, and flexible. As a leading member of the software team you will also engage in project management activities including but not limited to customer engagement, project task management, scheduling, and recruitment of new programming talent.
	- Requirements gathering from customers and R&D partners.
	- Developing and deploying shippable code for customer projects.
	- Maintaining a pulse on current software tools and trends in relevant areas of synthetic biology.
	- Collaborating with the Lattice development team to conceptualize, develop, and deploy state-of-the-art software solutions for synthetic biology.
	- Promoting and facilitating exposure of Lattice computational innovations to the synthetic biology community.
	- Creating, maintaining, and documenting a world-class codebase.
	- Managing development and deployment infrastructure.
	- You have a proven track record of designing and implementing innovative solutions to challenging computational problems (experience building scalable, reactive SPAs is a big plus).
	- Proficient knowledge of the current Javascript technologies landscape (ReactJS, Angular, and Typescript experience preferred).
	- Strong experience with data transformation and management technologies (GraphQL, REST, MongoDB, etc.)
	- Familiarity with package management tools (npm, yarn, etc.).
	- Exposure to server and deployment technologies (NodeJS, Docker, AWS, etc.).
	- Understanding of software engineering best practices and testing frameworks.
+ skill set:
	- You are passionate about synthetic biology and its potential. You are interested in developing computational approaches to synthetic biology and seeing them have an impact in real customer engagements. You are interested in furthering your education related to synthetic biology. You want to directly work with software engineers to see algorithms and technology directly translated into software artifacts. You are interested in engaging public audiences on the potential of Lattice approaches. You want to curate, maintain, and develop new and existing customer relationships. You are interested in working with lawyers to create material to help Lattice create patents and trademarks. You want to have a set of small internal software programming projects.
	- PhD and/or MBA in Computer Science/Engineering, Math, Bioengineering, or closely related Molecular Biology based field. Technical background preferred over business background.
	- Peer reviewed publications related to the development of computational approaches applied to biological problems. h-index greater than 10 a plus.
	- Experience with programming (Javascript, Java, Perl/Python preferred).
	- Experience developing algorithmic approaches to problems and evaluating their effectiveness.
	- Management experience a plus. You will likely have to oversee a team of 3-4 engineers.
	- Grant writing, reporting, and management a plus. Experience with SBIR awards also a plus.
	- Develop technical solutions to customer problems and communicate these to software engineering staff. This includes small internal software programming projects.
	- Pursue, develop, and maintain new customer engagements.
	- Pursue, develop, and maintain external funding sources (grants, awards, etc.).
	- Maintain high level of knowledge on current computational synthetic biology developments.
	- Work with customers to test Lattice software and translate customer feedback into engineering milestones.
	- Work with legal team to protect Lattice IP.
	- Promote Lattice technology, products, and process in public venues, conferences, etc.
+ skill set:
	- BLAH.
+ skill set:
	- BLAH.
+ skill set:
	- BLAH.
+ skill set:
	- BLAH.
+ skill set:
	- BLAH.
+ skill set:
	- BLAH.
+ skill set:
	- BLAH.
+ skill set:
	- BLAH.
+ skill set:
	- BLAH.
+ skill set:
	- BLAH.
+ skill set:
	- BLAH.
+ skill set:
	- BLAH.
+ skill set:
	- BLAH.
+ skill set:
	- BLAH.
+ skill set:
	- BLAH.
+ skill set:
	- BLAH.
+ skill set:
	- BLAH.
+ skill set:
	- BLAH.
+ skill set:
	- BLAH.
+ skill set:
	- BLAH.
+ skill set:
	- BLAH.
+ skill set:
	- BLAH.
+ skill set:
	- BLAH.
+ skill set:
	- BLAH.
+ skill set:
	- BLAH.
+ skill set:
	- BLAH.
+ skill set:
	- BLAH.
+ skill set:
	- BLAH.
+ skill set:
	- BLAH.
+ skill set:
	- BLAH.
+ skill set:
	- BLAH.
+ skill set:
	- BLAH.














## R&D roles in Bio Manufacturing Automation






Skills for R&D roles in bio manufacturing automation () cloud robot lab
+ skill set:
	- Architect the overall automation software, hardware, wetware ecosystem including hardware configurations, experimental protocols, and software control and scheduling systems.
	- Mentor other automation engineers.
	- Design, create, and deploy software to schedule, control, and manage the automation system.
	- Design, configure, and assemble a state-of-the-art automation system for biological engineering.
	- Work with synthetic biologists to translate manual protocols into automated processes and validate them at high throughput.
	- Work with hardware engineers to develop custom automation hardware to physically perform synthetic biology's experimental procedures.
	- Work with software engineers to develop custom software tools to schedule, control, and manage the automation system.
	- Develop metrics to measure and track the success of the automation platform.
	- Partner with automation vendors to vet software platforms and work to integrate them into our larger pipeline though the development of new APIs and methods.
	- Partner with automation vendors to obtain new hardware, develop new prototype hardware elements, build interfaces, and integrate hardware into the automation platform.
	- Embrace ambiguity and drive for impact.
	- Contribute to the company's product, direction, and culture.
	- B.S. or higher degree in Bioengineering, Computer Science or a related field. PhD preferred.
	- 5+ years direct experience in biotechnology industry. 2+ years directly with automation.
	- Strong coding skills in modern software development environments. GitHub (or similar) portfolio of work ideal.
	- Publication or patent record related to biotechnology.
	- Strong written and oral communication skills.
	- Ability to physically relocate to Cambridge, Massachusetts.
	- You are a highly skilled programmer with years of experience in industry or research. You want to work on software development projects with strong algorithmic and research foundations. You are highly skilled technical lead in laboratory automation with years of experience in industry or research. You are passionate about joining an early stage startup where autonomy, passion to learn, and excitement to engineer biology take precedence over process and ego. You ideally have some laboratory automation and biology wetlab experience. You are interested in learning more about how to integrate hardware, software, and wetware.
+ skill set:
	- Design, configure, and assemble a state-of-the-art automation system for biological engineering.
	- Work with synthetic biologists to translate manual protocols into automated processes and validate them at high throughput.
	- Work with software engineers to develop custom software tools to schedule, control, and manage the automation system.
	- Develop metrics to measure and track the success of the automation platform.
	- Partner with automation vendors to obtain new hardware, develop new prototype hardware elements, build interfaces, and integrate hardware into the automation platform.
	- Embrace ambiguity and drive for impact.
	- Contribute to the company's product, direction, and culture.
	- You are a highly skilled technical lead in laboratory automation with years of experience in industry or research. You are passionate about joining an early stage startup where autonomy, passion to learn, and excitement to engineer biology take precedence over process and ego. You ideally have some software programming and biology wetlab experience. You are interested in learning more about how to integrate hardware, software, and wetware.
+ skill set:
	- 1 - 3 years of experience working in either a laboratory environment or regulated manufacturing industry (Pharmacuetical, Medical Device, Biotechnology, Automotive, etc.)
	- Experience working in a Good Manufacturing Practices/Good Laboratory Practices (GMP/GLP) environment
+ skill set:
	- BLAH.
+ skill set:
	- BLAH.
+ skill set:
	- BLAH.
+ skill set:
	- BLAH.
+ skill set:
	- BLAH.
+ skill set:
	- BLAH.
+ skill set:
	- BLAH.
+ skill set:
	- BLAH.
+ skill set:
	- BLAH.
+ skill set:
	- BLAH.
+ skill set:
	- BLAH.
+ skill set:
	- BLAH.
+ skill set:
	- BLAH.
+ skill set:
	- BLAH.
+ skill set:
	- BLAH.
+ skill set:
	- BLAH.
+ skill set:
	- BLAH.
+ skill set:
	- BLAH.
+ skill set:
	- BLAH.
+ skill set:
	- BLAH.
+ skill set:
	- BLAH.
+ skill set:
	- BLAH.
+ skill set:
	- BLAH.
+ skill set:
	- BLAH.
+ skill set:
	- BLAH.
+ skill set:
	- BLAH.
+ skill set:
	- BLAH.
+ skill set:
	- BLAH.
+ skill set:
	- BLAH.
+ skill set:
	- BLAH.
+ skill set:
	- BLAH.
+ skill set:
	- BLAH.
+ skill set:
	- BLAH.
+ skill set:
	- BLAH.
+ skill set:
	- BLAH.
+ skill set:
	- BLAH.
+ skill set:
	- BLAH.
+ skill set:
	- BLAH.
+ skill set:
	- BLAH.







##	Skill Sets for Bioinformatics






Skill sets for bioinformatics:
+ skill set:
	- Developing scalable bioinformatics algorithms, pipelines and tools to solve complex, distributed computing challenges
	- Providing statistics, scientific computing and data analysis support to research, product development and manufacturing operations
	- BS in bioinformatics or related field with 6-8 years industry experience or MS/PhD in bioinformatics or related field with 4-6 years industry experience
	- Advanced proficiency in Python, R or Rust in a Linux environment
	- Broad experience with NGS data analysis, manipulation and presentation (in Python or R)
	- Excellent data analysis and visualization skills (in Python, R and using tools like Tableau)
	- Excellent understanding of molecular biology, with an emphasis on DNA chemistry preferred
	- Proven ability to synthesize and present conclusions from complex analyses in a clear and visually compelling way
	- Familiarity with common tools and file formats for genome-scale interval data manipulation and analysis (e.g. bedtools, bedops, etc.)
	- Formal software development experience, the software development lifecycle, unit and functional testing (via py.test) and version control via Git
	- Familiarity with database (PostgreSQL, NoSQL and Redshift) design, implementation and efficient query construction
	- Experience building and using Docker containers
	- Familiarity with cloud-based computing environments, especially Amazon Web Services
	- Experience using workflow management tools (e.g. Airflow, Galaxy, Luigi)
	- Familiarity with web application development including building REST-ful interfaces
	- Familiarity with DNA analysis software (MacVector, VectorNTI, SnapGene)
+ skill set:
	- We have an opening for a bioinformatician in the Bedford lab at the Fred Hutch to work on genomic epidemiology and evolutionary analysis of pathogens such as SARS-CoV-2, seasonal influenza, and other emerging and endemic pathogens. This position will contribute to ongoing work for the Bedford lab and Nextstrain.
	- Nextstrain is an award-winning project for tracking infectious disease epidemics developed in collaboration with the Neher lab at the University of Basel. Nextstrain won the Open Science Prize in Feb 2017 and has been instrumental in analysis of the SARS-CoV-2 pandemic, Ebola outbreaks, Zika spread in the Americas and is used by the World Health Organization to aid in the process of influenza vaccine strain selection. The software we write to power all parts of Nextstrain—bioinformatics, visualizations, analysis pipelines, data management, and more—is entirely open-source and available to the public. We work with public health entities and scientists across the world, both formally and informally, to expand pathogen surveillance capabilities and to improve the automation and robustness of these analyses. Our goal is to empower the wider genomic epidemiology and public health communities to tweak our analyses, create new ones, and communicate scientific insights using the same tools we do.
	- An ideal candidate would be local to the Seattle area or willing to relocate, but remote work is also an option.
	- This role advances the research aims of the Bedford lab and the Nextstrain team through a combination of independent work, collaboration with scientists and software developers in the group, and interactions with the wider public health and science communities.
	- Develop and maintain analytic pipelines such as those that clean and ingest genome metadata, build phylogenetic trees, and run forecasting models for SARS-CoV-2 and other pathogens
	- Improve the robustness, automation, and monitoring of our existing pathogen pipelines
	- Develop reproducible pipelines to expand surveillance of endemic and emerging human pathogens, in collaboration with both internal and external groups
	- Participate in community outreach through office hours, discussion forums, and mailing lists
	- Write and maintain thorough documentation on software and pipelines
	- Design software with a diverse range of collaborators and users in mind
	- Contribute to the Nextstrain team's decision-making and planning processes
	- Present at Bedford lab meetings
	- Master’s degree in bioinformatics, computational biology, biology, or related field with at least three years’ direct experience in computational analysis of large sequence-based molecular data sets.
	- Fluency in at least one high-level programming language, such as Python, R, Ruby, JavaScript, or Perl
	- Familiarity with version control and other software development best practices
	- Experience with workflow managers such as Snakemake, Nextflow, or WDL
	- Knowledge of molecular biology
	- Motivated to learn new skills and technologies and collaborate within an existing team’s practices
	- Excellent written and verbal communication skills
	- Expertise in genomics
	- Knowledge of automated testing and workflows such as GitHub Actions
	- Experience configuring and deploying analyses on a cloud infrastructure
+ skill set:
	- Senior Bioinformatics Engineer
	- Support Ginkgo’s Biosecurity Teams with your technical expertise in bioinformatics
	- Design, build, test, deploy, and maintain bioinformatics pipelines
	- Interface with software engineers to create and implement NGS data analysis pipelines for production use
	- Collaborate cross-functionally with wet and dry-lab operators on experimental design and analysis to optimize new analytic approaches using NGS technologies
	- Effectively communicate scientific results to technical and non-technical audiences
	- Assist with shaping Ginkgo and Concentric’s overall direction as a company and support its evolving needs, as necessary
	- BS with 6+years OR MS with 4+ years OR PhD with 2+ year of experience
	- 2 years of relevant industry experience is required
	- Proficient in at least one software programming language (Python)
	- Proficient in workflow management software (Nextflow, Docker)
	- Proficient in pipeline development on AWS infrastructure
	- Proficient in best practices for collaborative software development (GitHub, version control systems, CI/CD, test-driven development, and good documentation habits)
	- First-hand technical experience in at least two of the following disciplines:  bioinformatics, computer science, mathematics, genome assembly & annotation, statistics, probabilistic modeling, machine learning, or quantitative modeling of biological systems
	- Demonstrated ability to meet the demands of multiple concurrent projects, and work efficiently in a fast-paced, high-growth environment
	- Ability to adapt to changing priorities and pivot as needed in a fast-paced environment
	- Superb communication skills, with the ability to effectively convey project plans and complex technical details to a wide range of internal teams, executive leadership, and external customers
	- Strong curiosity for, and comfort working in, areas of biology & biosecurity previously unknown to you (and, at times, unknown to your peers)
	- Experience analyzing and benchmarking outputs from multiple sequencing technologies: Illumina, Oxford Nanopore, PacBio, Hi-C
	- Direct sequencing and bioinformatics experience with varied pathogen types (e.g. viral, bacterial, fungal) or engineered organisms
	- Experience with many or all of the following:  Python, bash/unix scripting, Snakemake, Nextflow, airflow, CWL, relational databases (SQL), GraphQL, distributed computing (AWS/Google Cloud), Docker, software version control (git)
	- Data and Database Management: SQL, AWS, Snowflake
	- Technical program management skill, SCRUM, and agile methodologies
+ First-hand technical experience in at least two of the following areas: bioinformatics; sequence analysis; comparative genomics; phylogenetics; NGS analysis; statistics; probabilistic modeling; machine learning; population genetics; metabolic engineering; or quantitative modeling of biological systems
+ skill set:
	- We're looking for a Software Solution Engineer who will work closely with Ginkgo scientists to understand their processes and quickly prototype and build the tools they need to create these new organisms, conduct experiments, control lab robots, and analyze massive amounts of data. You'll collaborate with a team of talented and diverse engineers, product managers, experts in data analysis and bioinformatics, and, of course, your customers – the dozens of biologists and chemists at Ginkgo. To be successful at this role, you should have a strong background in biology and bioinformatics, combined with excellent software development skills.
	- Our tech stack includes AWS, Docker, Django, REST, GraphQL, React, MySQL, Postgres, Elastic Search, Airflow, and other modern tools and frameworks. Our languages of choice are Python and Javascript. We don't expect you to be an expert in every platform, but we do expect that you have a solid foundation in computer science and have no problem mastering the technologies that we use today (and will adopt tomorrow).
	- Design and implement software solutions that address the current needs of Ginkgo bioengineers. These solutions could be a microservice, standalone tool, user-facing web application, data pipeline, etc.
	- Collaborate with your product manager to define milestones and deliverables.
	- Work as a member of an agile team, attend scrum meetings, and review code produced by your peers.
	- Write automated tests for your solution and oversee its deployment.
	- Monitor the production environment to detect and address customers' issues.
	- Prototype new ideas, algorithms, and frameworks.
	- Desired Experience and Capabilities
	- Degree in computer science, biology, bioinformatics, or relevant field
	- Relevant internship or work experience
	- Working knowledge of next-generation sequencing and omics technologies
	- Practical understanding of scalable bioinformatics pipelines
	- Knowledge of distributed systems architecture
	- Eagerness to work hand-in-hand with Ginkgo's foundry team to architect the next generation of foundry for engineering organisms
+ skill set:
	- Our tech stack includes AWS, Docker, Django, REST, GraphQL, React, MySQL, Postgres, Elastic Search, Airflow, and other modern tools and frameworks. Our languages of choice are Python and Javascript. We don't expect you to be an expert in every platform, but we do expect that you have a solid foundation in computer science and have no problem mastering the technologies that we use today (and will adopt tomorrow). And, most importantly, while you may not have a background in biology, you must be passionate about Ginkgo's mission of making biology easier to engineer.
	- We're looking for a Senior Software Engineer who will design, develop, test, and deploy the software used by Ginkgo scientists to create these new organisms, conduct experiments, control lab robots, and analyze massive amounts of data. You'll collaborate with a team of talented and diverse engineers, product managers, experts in data analysis and bioinformatics, and, of course, your customers – the dozens of biologists and chemists at Ginkgo.
	- Design and implement software solutions that address current needs of Ginkgo bioengineers. These solutions could be a microservice, standalone tool, user facing web application, data pipeline, etc.
	- Collaborate with your tech lead and product manager to define milestones and deliverables.
	- Work as a member of an agile team, attend scrum meetings, and review code produced by your peers.
	- Write automated tests for your solution and oversee its deployment.
	- Monitor the production environment to detect and address customers' issues.
	- Prototype new ideas, algorithms, and frameworks.
	- BS, MS, or PhD in computer science with 5+ years of industry experience
	- Proven track record in full-stack development
	- Experience with building distributed systems
	- Experience with code management systems, CI/CD, AWS, or any other cloud experience a plus
	- Familiarity with biology a plus
	- Previous startup experience a plus
	- Eagerness to work hand-in-hand with the Ginkgo's foundry team to architect the next generation of foundry for engineering organisms









































##	Skill Sets for Computational Pharmacology



Skill sets for computational pharmacology:
+ skill set:
	- Scientist - Modeling and Simulation
	- The Scientist will report to the Senior Director and engage with clients on clinical pharmacology and pharmacometrics projects. The Scientist will assist in supporting drug development programs by applying clinical pharmacology and pharmacometric methodology from pre-clinical through late-stage development and regulatory filings.
	- Part of the Clinical Pharmacology, Modeling and Simulation team
	- Develop analysis plans, execute and communicate per standard operating practices of Pumas-AI.
	- Work closely with clients' teams.
	- Prepare reports, data and programs in a manner suitable for regulatory submissions.
	- Create and manage datasets and tasks such as non-compartmental and compartmental PK/PD analysis.
	- Collaborate with Pumas-AI pharmacometricians in the modeling of emerging PK and PK/PD data.
	- Maintain knowledge of relevant scientific and regulatory practices and trends and ensure that clinical pharmacology aspects of development programs are contemporary.
	- Scoping questions and developing modeling and simulation strategy for projects.
	- Analysis and reporting of population pharmacokinetic/pharmacodynamic, exposure-response, and disease progression models to support dose selection, trial design, and Go/No-Go decisions.
	- Participate in the preparation of reports/abstracts/manuscripts for publications and regulatory documents.
	- Prepare formal presentations and written reports to Pumas-AI standards.
	- Carry out functional responsibilities in accordance with applicable SOPs and Regulatory requirements
	- A PhD or a relevant postdoctoral training in Pharmaceutical Sciences, Clinical Pharmacology, Biological Sciences, Bioengineering, Chemical Engineering, or Applied Mathematics.
	- Master’s degree and 4 years of industry or relevant academic experience in applied statistics/mathematics, biostatistics, engineering, or pharmaceutical sciences.
	- Bachelor’s degree and 6 years of industry or relevant academic experience in applied statistics/ mathematics, biostatistics, engineering, or pharmaceutical sciences.
	- Demonstrated ability to use quantitative tools.
	- Excellent communication skills and ability to work independently and as a team member.
	- Hands-on experience in at least one of the modeling and simulation software.
	- Ability to multi-task and prioritize assignments, establishing courses of action for self and others to ensure that work is completed efficiently in a timely manner.
	- Must be self-motivated, well-organized, detail-oriented and have excellent written and verbal communication skills.
	- Experience in establishing collaborative working relationships with internal and external stakeholders.
	- Pumas-AI is a young company led by decades of experience in diverse areas of healthcare and mathematical modeling. With a focused goal on providing better solutions for patient care, be it via pharmaceutical drug development innovation or tools for point of care, Pumas-AI aims to bring value, innovation and efficiency to healthcare. We build technology that converts the power of science into real-world applications with unprecedented speed and accuracy. At the core of everything we do is our commitment to move science forward.


























##	Applied Machine Learning Roles in the Biotechnology & Pharmaceutical Industry








+ tech stack:
	- You should think about joining us if you care about making a difference in treating disease and saving human lives. But also, if you are up to tackling some of the hardest open challenges in deep learning today:
		* Non-stationary, unbalanced, and noisy data: Our training data is seldom i.i.d.; new medicines are unlocked by pushing out into newly-discovered biology. Classes are extremely unbalanced, ratios of 1 positive to 70,000 negatives are typical. Help us reason about how to learn appropriately without dismissing nor overfitting to the data; identify when we can trust a label or have confidence in a prediction; and develop techniques to find and correct for systemic biases.
		* Extreme scaling: Medicinal chemists can synthesize about a trillion trillion molecules today. Help us scale predictive algorithms to orders of magnitude beyond those contemplated in any other problem domain today.
		* Multi-parameter optimization: Medicine has to be both safe and effective, so we have to concurrently optimize a number of criteria such as potency, selectivity, solubility, toxicity, synthesizability, etc. Help us efficiently explore the Pareto frontier and avoid mode collapse.
		* Adversarial generation of synthetic data: Data augmentation has shown utility in improving the robustness of predictions. Help us find ways to best integrate molecular physics simulation and machine learning to impute new data.
		* Explainability and visualization: Subtle patterns govern molecular recognition. Help us to understand how they lead to the discovery of fundamental chemistry by AI.
	- Ph.D. or M.Sc. in computer science, statistics, data science, or related field
	- 5+ years of extensive practical experience and proven track record of developing, implementing, debugging, and extending machine learning algorithm
	- Knowledge of modern neural network frameworks such as Tensorflow, Torch, or Theano
	- Strong analytical and statistical skills
	- Scientific rigor, healthy skepticism, and detail-orientation in running and analyzing experiments
	- Familiarity with processing large data sets in a Linux environment
	- Software engineering skills and coding experience in at least one high-level programming language (Python, R, Java, C/C++, etc.)
	- Biomedical knowledge or experience in processing chemical or biological data is preferred but not required
	- Experience with cloud computing environments (AWS/Azure/GCE)
	- https://www.atomwise.com/jobs/senior-machine-learning-research-scientist/
+ skill set:
	- AI/ML Engineer - DNA & RNA design
	- Our mission is to make biology easier to engineer. Ginkgo is constructing, editing, and redesigning the living world in order to answer the globe’s growing challenges in health, energy, food, materials, and more. Our bioengineers make use of an in-house automated foundry for designing and building new organisms. 
	- The Design Team’s mission is to unlock the power of synthetic biology by making genetic engineering — the art of designing genotypes and underlying DNA that ultimately achieves a desired phenotype — more predictive and robust. We are an interdisciplinary team with core expertise in computational biology, genomics, bioinformatics, microbial and mammalian genetics, RNA biology, statistics, and software engineering. We design and develop GInkgo’s genetic design platform, and directly support Ginkgo’s portfolio of customer projects with design and analysis of high-throughput experiments. We work in a broad range of application areas, including health, agriculture, specialty chemicals, and biosecurity.
	- Ginkgo recently announced a large partnership with Google Cloud to build a generative AI platform for engineering biology and for biosecurity. The Design Team is responsible for delivering best-in-class foundation and application specific models for design, analysis, and functional prediction of nucleic acid sequences, organismal phenotypes, and more. 
	- As a Machine Learning / Artificial Intelligence (ML/AI) Engineer on Ginkgo’s Design Team, you will leverage Ginkgo’s wealth of proprietary sequence and experimental data to design, train, and experimentally validate novel foundation and application-specific (e.g. fine-tuned) AI models for application to classification and design of genes, regulatory elements, multi-gene pathways, and even genomes. In addition, you will collaborate with experts throughout the organization to identify transformational opportunities for application of AI in genetic design, as well as in analysis of phenotypic data.
	- We have access to nearly limitless compute capacity: CPU, GPU, or TPU. In addition, you will have access to Ginkgo's experimental platform and resulting large datasets for generating and testing hypotheses using machine learning approaches as well as informing strategic training data acquisition.
	- The successful candidate will bring practical experience in modern AI/ML methods, creativity in solving problems at the intersections of scientific domains, a collaborative mindset, and enthusiasm for professional growth. We are looking for someone who is equally comfortable dreaming big as rolling up their sleeves and digging through the weeds.
	- Foundation Model (FM) development: Conceive, develop, and validate best-in-class foundation models for DNA and/or RNA, leveraging Ginkgo’s large and diverse proprietary sequencing datasets. Collaborate in cross-functional teams to design and experiment with model architectures, data models, data representations.
	- Application-specific model development: Conceive, develop, and validate purpose-built models (e.g. fine-tuned) for a range of DNA and RNA design and prediction applications
	- Influence strategic dataset acquisition: Partner with world-class experimentalists and hundreds of robots to conceive and design experiments to collect high-value training data at unprecedented scale. Influence how routine experiments are performed to maximize future learning potential. 
	- Influence the roadmap for AI at Ginkgo: Identify opportunities for application of AI and ML across the company, create prototypes, and contribute to overall prioritization and roadmap development for AI at Ginkgo.
	- Maintain high-quality documentation of your work and discoveries, creating written reports, technical presentations for internal or external audiences, electronic lab notebooks, internal database records, code comments, and software documentation.
	- Take part in something big: This is a growing team, a significant company focus, and a rapidly evolving field. You will be able to influence where things go and how they change.
	- Minimum Requirements
		* Bachelor’s degree and 4+ years, or Master’s degree and 2+ years, of subsequent ***experience in applying AI/ML to biological applications***
		* ***Subject matter expertise in genetics, genomics, transcription, translation, or RNA biology***
		* ***Familiarity with recent literature and state of the art for large model architectures and training approaches***
		* Deep ***knowledge of AI/ML literature as applied to DNA or RNA***
		* Proficiency with at least one programing language (Python preferred)
		* ***Experience with building machine/deep learning models with at least one common framework such as PyTorch, Tensorflow, or JAX
	- Preferred Capabilities and Experience***
		* PhD in artificial intelligence, computer science, synthetic biology, computational biology, genomics, bioinformatics, quantitative biology, chemical engineering, or another related field. Interdisciplinary work is strongly preferred.
		* ***Hands-on experience (3+ years) in developing deep neural networks. Deep knowledge of currently available AI model architectures and data schemes. Perspective on advantages and drawbacks of various approaches***
		* ***Broad knowledge of state-of-art machine learning approaches to biological sequence analysis***
		* ***Significant hands-on experience in using software libraries such as tensorflow, pytorch, jax, and keras for model construction***
		* ***Exposure with ML and data orchestration and workflow engines like Airflow, Kubeflow, Flyte, or Dagster.***
			+ Flyte
				- https://flyte.org/
				- Build & deploy data & ML pipelines, hassle-free
				- The infinitely scalable and flexible workflow orchestration platform that seamlessly unifies data, ML and analytics stacks.
				- Create extremely flexible data and ML workflows
				- End-to-end data lineage
				- Collaborate with reusable components
				- Integrate at the platform level
				- Allocate resources dynamically
			+ Dagster
				- https://dagster.io/
				- Cloud-native orchestration of data pipelines
				- Ship data pipelines with extraordinary velocity
				- The cloud-native orchestrator for the whole development lifecycle, with integrated lineage and observability, a declarative programming model, and best-in-class testability.
				- Manage your data assets with code
				- An orchestration platform for the development, production, and observation of data assets.
		* Expertise in best practices for software development, including version control, code reviews, unit testing, and continuous integration. ***Experience in ML model management, MLOps, is a plus***
		* ***Proven track record of delivering in cross-functional teams and in project management. Excellence in scientific communication***
		* ***Enthusiasm to learn new techniques. Strong curiosity of areas of biology previously unknown to you***
	- Total compensation for this role is market driven, with a starting salary of $90K+, as well as company stock awards.
+ skill set:
	- Senior ML (Machine Learning) Engineer
	- Our mission is to make biology easier to engineer. Ginkgo is constructing, editing, and redesigning the living world in order to answer the globe’s growing challenges in health, energy, food, materials, and more. Our bioengineers make use of an in-house automated foundry for designing and building new organisms. 
	- Ginkgo recently announced a large partnership with Google Cloud to build a generative AI platform for engineering biology and for biosecurity. The AI Enablement team is responsible for delivering the ML expertise required to make this happen. We have an ambitious goal: making it as easy as possible to use AI and ML across all of Ginkgo. We have access to nearly limitless compute capacity: CPU, GPU, or TPU. We partner closely with biologists, software and DevOps engineers, and data scientists to create the necessary ML infrastructure, organize data, invent, evaluate, train, and fine-tune new models and approaches, consult and prototype new applications - whatever needs doing!
	- As one of the first members of this new team, you will have a large role in establishing the core architecture, making technical choices, and creating patterns of our model training and serving infrastructure. You will help create the data and model provenance and management infrastructure necessary for such a project.
	- You will work closely with experts in DNA design and Protein Engineering to enable scientists of various levels of exposure to modern ML to create foundation models for biology, train new task-specific models, and leverage existing ones.
	- On occasion, you may need to do some data archaeology, quickly learn the basics of protein folding or codon optimization, become the company’s expert on a new tool, or pair with your VP to work out capacity planning. You will identify what needs to happen, bring it to the team’s attention, and make it happen.
	- You will not be expected to be an expert in All The Things. You will be expected to have a high level of general technical competency, be a fast learner brimming with curiosity, and an expert in a few things. 
	- Build, manage, and evolve a GCP-based platform for large scale  (up to 100B+ parameters) training, evaluation, and serving of Foundation Models for biology. 
	- Develop, implement and maintain a system for creating smaller models that combine large FMs with additional experimental data to address specific needs and applications.
	- Own processes for data ingestion, data prep, data and model provenance tracking, and various other data engineering and ML Ops activities.
	- Contribute to model design and experimentation.
	- Identify opportunities for application of AI and ML across the company, create prototypes, and contribute to overall prioritization and roadmap development for AI at Ginkgo.
	- Minimum Requirements
		* PhD in a scientific discipline and a minimum of 5 years related experience; may include post-doctoral experience; Masters and 7 years of related experience; Bachelors and 9 years of related experience in data engineering, systems engineering, machine learning and operations, MLOps, or similar roles; or equivalent industry experience.
		* Deep experience with Python.
		* Experience with ML and data orchestration and workflow engines like Airflow, Kubeflow, Flyte, or Dagster.
		* Familiarity with recent literature and state of the art for large model architectures and training approaches
		* Experience with building machine/deep learning models with at least one common framework such as PyTorch, Tensorflow, or JAX.
	- Preferred Capabilities and Experience
		* Practical experience iterating on LLM design 
		* Familiarity with the ML ecosystem, including MLFlow and related tools
		* Experience with Terraform or Pulumi, Kubernetes
			+ Pulumi - Infrastructure as Code in Any Programming Language
			+ https://www.pulumi.com
			+ Pulumi's open source infrastructure as code SDK enables you to create, deploy, and manage infrastructure on any cloud, using your favorite languages.
			+ Automate your infrastructure at any scale
			+ Your Code, Your Cloud, Your Success
			+ Build, manage, and scale your infrastructure, secrets, and configurations in one place.
		* Experience operating non-trivial GCP deployments
		* Experience with Vertex AI services
		* Experience with “Cloud Life Sciences” / Google Batch
	- Total compensation for this role is market driven, with a starting salary of $150K +, as well as company stock awards. 
+ skill set:
	- Principal Machine Learning Researcher
	- Our mission is to make biology easier to engineer. Ginkgo is constructing, editing, and redesigning the living world in order to answer the globe’s growing challenges in health, energy, food, materials, and more. Our bioengineers make use of an in-house automated foundry for designing and building new organisms. 
	- Ginkgo recently announced a large partnership with Google Cloud to build a generative AI platform for engineering biology and for biosecurity. The AI Enablement team is responsible for delivering the ML expertise required to make this happen. We have an ambitious goal: making it as easy as possible to use AI and ML across all of Ginkgo. We have access to nearly limitless compute capacity: CPU, GPU, or TPU. We partner closely with biologists, software and DevOps engineers, and data scientists to create the necessary ML infrastructure, organize data, invent, evaluate, train, and fine-tune new models and approaches, consult and prototype new applications - whatever needs doing!
	- As one of the first members of this new team, you will have a large role in molding our approaches to creating foundation models for biology, as well as creating fine-tuned and derived models for specific applications in bioengineering. You will work closely with experts in DNA design and Protein Engineering to achieve this goal.
	- You will be the go-to expert for deep learning model architectures and training approaches. You will often take the lead on novel model design and evaluation. Many other types of work will come your way. You may need to do data archaeology, create and debug pipelines in tools like Kubeflow or Flyte, quickly learn the basics of protein folding or codon optimization, become the company’s expert on a new tool, debug odd results created by a production model for a project under a time crunch, teach seminars and deliver tech talks, or pair with an engineer to improve the scaling characteristics of our key training pipelines. 
	- This is a new team, a significant company focus, and a rapidly evolving field. You will need to be able to handle ambiguity and uncertainty; on the flip side, you will be able to influence where things go and how they change.
	- You will identify what needs to happen, bring it to the team’s attention, and make it happen.
	- You will not be expected to be an expert in All The Things. You will be expected to have a high level of general technical competency, be a fast learner brimming with curiosity, and an expert in a few things - deep learning, in particular. 
	- Lead design and evaluation of new foundation models for biology
	- Establish patterns for creating derived / fine-tuned models for specific biological applications
	- Leverage Ginkgo’s extensive internally generated data, and public and in-licensed datasets of protein and DNA sequences, assay results, fermentation data time series, research archives, lab notebooks, and more. 
	- Create processes for appropriate data featurization, model fine-tuning, benchmarks and evaluations, etc.
	- Work collaboratively with a team of engineers and scientists
	- Provide guidance, document key decisions, serve as go-to expert in Deep Learning
	- Identify opportunities for application of AI and ML across the company, create prototypes, and contribute to prioritization and roadmap development for AI at Ginkgo
	- Share results and learnings through means such as whitepapers, blog posts, workshops, or presentations
	- Ph.D in Artificial Intelligence, Computer Science, or equivalent (extensive practical experience may be considered in lieu of degree)
	- Eight or more years of experience in industry
	- Deep experience with Python.
	- Expert knowledge of recent literature and state of the art for large model architectures and training approaches
	- Extensive experience with building and iterating on machine/deep learning models using common frameworks such as PyTorch, Tensorflow, or JAX. Demonstrated expertise in implementing and fine-tuning neural networks, including design choices such as activation functions, loss functions, and regularization techniques.
	- Excellent ability to explain complex technical concepts, in writing as well as verbally
	- Experience with data preprocessing techniques, including feature engineering and data augmentation.
	- Ability to collaborate effectively with cross-functional teams, including data scientists, engineers, and domain experts.
	- Proficiency in using cloud computing platforms and distributed systems for training large-scale machine learning models.
	- Familiarity with the ML ecosystem and ability to explain pros and cons of various options
	- Understanding of explainability and interpretability in machine learning models, and experience with techniques like feature importance analysis or model-agnostic methods.
	- Familiarity with automated machine learning (AutoML) tools and frameworks like H2O.ai or Google Cloud AutoML.
	- Experience with ML and data orchestration and workflow engines like Airflow, Kubeflow, Flyte
	- Experience with deploying machine learning models in production environments using containerization and deployment tools like Docker or Kubernetes.
	- Familiarity with Terraform and other standard DevOps tooling
	- Understanding of cloud services and platforms for machine learning and big data processing. GCP / Vertex AI experience.
	- Experience operating in non-trivial cloud deployments
	- Exposure to computational biology, bioinformatics, protein engineering, DNA synthesis, & more: very welcome but not required
	- Total compensation for this role is market driven, with a starting salary of $170K+, as well as company stock awards. 
+ skill set:
	- AI/ML Engineer, Therapeutic Protein Design
	- Our mission is to make biology easier to engineer. Ginkgo is constructing, editing, and redesigning the living world in order to answer the globe’s growing challenges in health, energy, food, materials, and more. Our bioengineers make use of an in-house automated foundry for designing and building new organisms. 
	- The Protein Engineering Team works to address the complex challenges of protein and enzyme discovery, optimization, and design. We develop, onboard, stress-test, and utilize state-of-the-art protein discovery and design techniques in a continuously improving suite of AI, machine learning, and computational protein design tools. We are responsible for building a platform to “make proteins easier to engineer”, and deliver on protein design challenges that span a range of applications from health, agriculture, commodity chemicals, materials, flavors and fragrances, and more. 
	- Ginkgo recently announced a large partnership with Google Cloud to build a generative AI platform for engineering biology and for biosecurity. The Protein Engineering team is responsible for delivering best-in-class foundational models and application-specific models to tackle a range of protein design challenges, from generative models for design of enzyme sequences with novel catalytic functions to developing models for significantly accelerating the development of therapeutic proteins. 
	- As a Therapeutic Protein Engineer at Ginkgo Bioworks, you will play a vital role in advancing our efforts to engineer and optimize therapeutic proteins for diverse applications. Your expertise in protein design, coupled with your proficiency in machine learning and AI, will enable you to drive the development of novel therapeutic proteins with improved or novel functional characteristics in addition to engineering for developability properties such as solubility, stability, and reduced immunogenicity. 
	- You will leverage Ginkgo’s wealth of proprietary protein data to design, build, and train novel machine learning and AI models for application to protein design and optimization. We have access to nearly limitless compute capacity: CPU, GPU, or TPU. In addition, you will have access to Ginkgo's experimental platform and resulting large datasets for generating and testing hypotheses using machine learning approaches as well as informing strategic training data acquisition.
	- Protein Design and Engineering: Lead and contribute to the design and engineering of therapeutic proteins, focusing on enhancing developability attributes such as solubility, stability, prolonged half-life in serum, formulation considerations, and mitigating immunogenicity through computational methods.
	- Machine Learning and AI Integration: Develop and implement machine learning and AI algorithms to predict and optimize protein properties, allowing for the creation of highly effective and safe therapeutic proteins.
	- Immunogenicity Mitigation: Apply deep knowledge of immunogenicity mechanisms to design strategies that effectively mitigate immunogenic responses in therapeutic proteins, ensuring their safety and efficacy.
	- Data Analysis: Analyze extensive biological datasets to extract insights into protein engineering and developability, informing decision-making processes in therapeutic protein development.
	- Collaboration: Collaborate closely with interdisciplinary teams of scientists, including biochemists, mammalian biological engineers, and other protein designers to translate computational insights into practical therapeutic solutions.
	- Research and Development: Stay updated with the latest advancements in protein design, machine learning, and AI through active participation in scientific literature review and attendance at relevant conferences.
	- Influence the roadmap for AI at Ginkgo: Identify opportunities for application of AI and ML across the company, create prototypes, and contribute to overall prioritization and roadmap development for AI at Ginkgo.
	- Take part in something big: This is a growing team, a significant company focus, and a rapidly evolving field. You will be able to influence where things go and how they change.
	- PhD in bioengineering, computer science, biophysics, biochemistry, computational biology, bioinformatics, chemical engineering, or related field plus 3 years experience. May include post doctoral or industry experience. Masters plus 5 years additional postgraduate or industry relevant experience. 
	- Hands-on experience (5+ years) in protein design and engineering, with a deep understanding of developability considerations for therapeutic proteins
	- Hands-on experience (2+ years) in developing and applying machine learning and AI algorithms for protein design
	- Broad knowledge of state-of-art machine learning approaches to protein design and biological data analysis. Familiarity with recent literature and state of the art for large model architectures and training approaches
	- Fluency with Python. Knowledge of best practices for software development, including version control, code reviews, unit testing, and continuous integration
	- Strong communication skills, both written and verbal, for effective collaboration within interdisciplinary teams
	- Enthusiasm to learn new techniques. Strong curiosity of areas of biology previously unknown to you
	- Ability to work independently, manage multiple projects, and meet project timelines
	- Strong publication record in protein design and engineering for therapeutic protein applications
	- 4+ years experience in applying protein design to any application within protein binder engineering, bispecific antibody design, CAR-T / cell therapy applications, gene editors, or capsid engineering 
	- Knowledge and hands-on experience with physics-based protein modeling and design methods are a plus. Familiarity with at least one type of molecular modeling software such as PyMOL, Rosetta, Schrodinger, Molecular Operating Environment (MOE) a plus. Expertise in the computational modeling of biomacromolecules is a plus
	- Hand-on wet-lab experience in protein sciences or experience working closely with experimentalists strongly preferred
	- Total compensation for this role is market driven, with a starting salary of $110K +, as well as company stock awards.
+ skill set:
	- AI/ML Engineer, Protein Design
	- Our mission is to make biology easier to engineer. Ginkgo is constructing, editing, and redesigning the living world in order to answer the globe’s growing challenges in health, energy, food, materials, and more. Our bioengineers make use of an in-house automated foundry for designing and building new organisms. 
	- The Protein Engineering Team works to address the complex challenges of protein and enzyme discovery, optimization, and design. We develop, onboard, stress-test, and utilize state-of-the-art protein discovery and design techniques in a continuously improving suite of AI, machine learning, and computational protein design tools. We are responsible for building a platform to “make proteins easier to engineer”, and deliver on protein design challenges that span a range of applications from health, agriculture, commodity chemicals, materials, flavors and fragrances, and more. 
	- Ginkgo recently announced a large partnership with Google Cloud to build a generative AI platform for engineering biology and for biosecurity. The Protein Engineering team is responsible for delivering best-in-class foundational models and application-specific models to tackle a range of protein design challenges, from generative models for design of enzyme sequences with novel catalytic functions to developing models for significantly accelerating the development of therapeutic proteins.
	- As a Machine Learning / Artificial Intelligence (ML/AI) Protein Engineer, you will leverage Ginkgo’s wealth of proprietary protein data to design, build, and train novel foundational and application-specific (e.g. fine-tuned) AI models for application to protein design and optimization. We have access to nearly limitless compute capacity: CPU, GPU, or TPU. In addition, you will have access to Ginkgo's experimental platform and resulting large datasets for generating and testing hypotheses using machine learning approaches as well as informing strategic training data acquisition. 
	- We are looking for someone who is excited about the promise of artificial intelligence in synthetic biology and the premier role of biomolecular engineering in biology by design. If you sleep not only to maintain healthy levels of protein in the brain, but also dream of harnessing the power of machine learning and artificial intelligence to design enzymes and other proteins, then you are at the right place.
	- While this is a fully remote position, you will be expected to travel approximately every six months for company on-sites. There may also be infrequent travel where required with advanced notice and discussion, where appropriate.  
	- Foundational Model (FM) development: Conceive and develop best-in-class foundational protein language models leveraging Ginkgo’s large and diverse proprietary protein datasets. Collaborate in cross-functional teams to design and experiment with model architectures and data schemes.
	- Application-specific model development: Conceive and develop purpose-built models (e.g. fine-tuned) for a range of protein design applications leveraging Ginkgo’s unprecedented scale of protein experimental data 
	- Influence strategic dataset acquisition: Partner with world-class experimentalists and hundreds of robots to conceive and design experiments to collect high-value training data at unprecedented scale. Influence how routine experiments are performed to maximize future learning potential. 
	- Influence the roadmap for AI at Ginkgo: Identify opportunities for application of AI and ML across the company, create prototypes, and contribute to overall prioritization and roadmap development for AI at Ginkgo.
	- Take part in something big: This is a growing team, a significant company focus, and a rapidly evolving field. You will be able to influence where things go and how they change.
	- PhD in bioengineering, computer science, math, biophysics, computational biology, bioinformatics, chemical engineering, or related field. May include post doctoral or industry experience. Masters plus 5 years additional postgraduate or industry relevant experience. 
	- Hands-on experience (3+ years) in developing large foundation models or fine-tuned applications. Deep knowledge of currently available AI model architectures and data schemes. Perspective on advantages and drawbacks of various approaches. 
	- Broad knowledge of state-of-art machine learning approaches to protein design and biological data analysis. Familiarity with recent literature and state of the art for large model architectures and training approaches
	- Fluency with Python. Expertise in best practices for software development, including version control, code reviews, unit testing, and continuous integration. Experience in ML models management, MLOps, is a plus.
	- Significant hands-on experience in developing software libraries such as tensorflow, pytorch, jax, keras, and scikit for model construction.
	- Experience with building machine/deep learning models with at least one common framework such as PyTorch, Tensorflow, or JAX. 
	- Strong communication skills, both written and verbal, for effective collaboration within interdisciplinary teams.
	- Enthusiasm to learn new techniques. Strong curiosity of areas of biology previously unknown to you.
	- Ability to work independently, manage multiple projects, and meet project timelines.
	- Strong publication record in developing and applying AI/ML to protein design and engineering applications 
	- Some exposure with ML and data orchestration and workflow engines like Airflow, Kubeflow, Flyte, or Dagster.
	- Knowledge and hands-on experience with physics-based protein modeling and design methods are a plus. Familiarity with at least one type of molecular modeling software such as PyMOL, Rosetta, Schrodinger, Molecular Operating Environment (MOE) a plus. Expertise in the computational modeling of biomacromolecules is a plus.
	- Hand-on wet-lab experience in protein sciences or experience working closely with experimentalists strongly preferred 
	- Total compensation for this role is market driven, with a starting salary of $110K+, as well as company stock awards.
+ skill set:
	- BLAH.
+ skill set:
	- BLAH.
+ skill set:
	- BLAH.
+ skill set:
	- BLAH.
+ skill set:
	- BLAH.
+ skill set:
	- BLAH.
+ skill set:
	- BLAH.
+ skill set:
	- BLAH.
+ skill set:
	- BLAH.
+ skill set:
	- BLAH.
+ skill set:
	- BLAH.
+ skill set:
	- BLAH.
+ skill set:
	- BLAH.
+ skill set:
	- BLAH.
+ skill set:
	- BLAH.
+ skill set:
	- BLAH.
+ skill set:
	- BLAH.
+ skill set:
	- BLAH.
+ skill set:
	- BLAH.
+ skill set:
	- BLAH.
+ skill set:
	- BLAH.
+ skill set:
	- BLAH.
+ skill set:
	- BLAH.
+ skill set:
	- BLAH.
+ skill set:
	- BLAH.
+ skill set:
	- BLAH.
+ skill set:
	- BLAH.
+ skill set:
	- BLAH.
+ skill set:
	- BLAH.
+ skill set:
	- BLAH.
+ skill set:
	- BLAH.
+ skill set:
	- BLAH.
+ skill set:
	- BLAH.
+ skill set:
	- BLAH.
+ skill set:
	- BLAH.






##	Data Science Roles in the Biotechnology & Pharmaceutical Industry




+ Ideal candidate would have experience with metabolomic datasets and familiarity with analytical chemistry/assay methods, automation systems, and LIMs
	- LIMs???
		* for lab data management
		* LIMS: laboratory information management systems
			- or, LIMs: laboratory information management systems
			- or, LIS: laboratory information systems
			- or, LMS: laboratory management systems
+ skill set:
	- Experience in at least one of the following sequencing methodologies: cell-free DNA sequencing, methylation sequencing, single cell sequencing
	- Deep understanding of next generation sequencing methods, platform-specific bias and errors, and data interpretation
	- Hands-on experience with large-scale genomic data (e.g. whole genome sequencing, exome-seq, target enrichment, amplicon sequencing, RNA-seq, etc.)Hands-on experience with genome alignment, mapping, variant calling, and annotation tools
	- Familiarity with human and mammalian genomics, standard genomic databases (UCSC, Ensembl, NCBI), formats, and tools.
	- Proficiency in python, ruby, Perl, or another high-level scripting language (Python highly preferred)
	- Familiarity with UNIX-based operating systems and shell scripting
	- Excellent data analysis and visualization skills (Tableau, R, matplotlib, or similar)
	- Excellent understanding of molecular biology and cell biology
	- Experience with Seven Bridges, DNA Nexus or similar platform
+ skill set:
	- M.S. or Ph.D. degree in Genetics, Biology, Bioinformatics, Biostatistics, Computational Biology, Computer Science, or related field
	- Knowledge and experience in the field of cancer or rare disease genomics.
	- Experience in at least one of the following sequencing methodologies: cell-free DNA sequencing, methylation sequencing, single cell sequencing
	- Deep understanding of next generation sequencing methods, platform-specific bias and errors, and data interpretation
	- Hands-on experience with large-scale genomic data (e.g. whole genome sequencing, exome-seq, target enrichment, amplicon sequencing, RNA-seq, etc.)
	- Hands-on experience with genome alignment, mapping, variant calling, and annotation tools
	- Familiarity with human and mammalian genomics, standard genomic databases (UCSC, Ensembl, NCBI), formats, and tools.
	- Proficiency in python, ruby, Perl, or another high-level scripting language (Python highly preferred)
	- Familiarity with UNIX-based operating systems and shell scripting
	- Excellent data analysis and visualization skills (Tableau, R, matplotlib, or similar)
	- Excellent understanding of molecular biology and cell biology
	- Strong interpersonal communication skills
	- Excellent publication track record
	- Additional desired qualities include:
		* Familiarity with version control (git, subversion)
		* Experience with target enrichment
		* Experience in clinical laboratory testing
		* Experience with Seven Bridges, DNA Nexus or similar platform
+ tech stack:
	- M.Sc/Ph.D. in Computer Science, Statistics, Cheminformatics, Bioinformatics, Computational Biology or B.S. in Computer Science with 7+ years experience.
	- Strong computer science fundamentals
	- 5+ years of experience in database engineering, data processing pipelines, and HPC
	- Strong database design and software-engineering best practices
	- Strong knowledge of statistics, data analytics, and data visualization
	- Strong coding skills in at least one high-level programming language (Python, Java, C++, etc)
	- Good familiarity with Linux command-line environment
	- Experience in bioinformatics or cheminformatics, working with ingestion of third-party and internal data sources
	- Experience working with scalable algorithms utilizing large amounts of data
	- Experience with cloud computing environments (AWS/Azure/GCE)
	- Experience with non-relational databases
	- Familiarity with organic chemistry
	- https://www.atomwise.com/jobs/senior-data-engineer-cheminformatics-bioinformatics/
+ skill set:
	- BLAH.
+ skill set:
	- BLAH.
+ skill set:
	- BLAH.
+ skill set:
	- BLAH.
+ skill set:
	- BLAH.
+ skill set:
	- BLAH.
+ skill set:
	- BLAH.
+ skill set:
	- BLAH.
+ skill set:
	- BLAH.
+ skill set:
	- BLAH.
+ skill set:
	- BLAH.
+ skill set:
	- BLAH.
+ skill set:
	- BLAH.
+ skill set:
	- BLAH.
+ skill set:
	- BLAH.
+ skill set:
	- BLAH.
+ skill set:
	- BLAH.
+ skill set:
	- BLAH.
+ skill set:
	- BLAH.
+ skill set:
	- BLAH.
+ skill set:
	- BLAH.
+ skill set:
	- BLAH.
+ skill set:
	- BLAH.
+ skill set:
	- BLAH.
+ skill set:
	- BLAH.
+ skill set:
	- BLAH.
+ skill set:
	- BLAH.
+ skill set:
	- BLAH.
+ skill set:
	- BLAH.
+ skill set:
	- BLAH.
+ skill set:
	- BLAH.
+ skill set:
	- BLAH.
+ skill set:
	- BLAH.
+ skill set:
	- BLAH.
+ skill set:
	- BLAH.
+ skill set:
	- BLAH.
+ skill set:
	- BLAH.
+ skill set:
	- BLAH.
+ skill set:
	- BLAH.
+ skill set:
	- BLAH.












#	Research Institutes for Ph.D.s in Biochemistry, Biology, Biotechnology, Medicinal Chemistry, and/or Pharmaceutical Science 










+ Fred Hutchinson Cancer Center
	- Combines Fred Hutchinson Cancer Research Center with Seattle Cancer Care Alliance (SCCA).
	- Seattle, WA.
	- Also known as:
		* Fred Hutch
		* The Hutch
	- https://www.fredhutch.org/en/about/careers.html







#	Research Intitutes/Centers at Research Universities



+ UCSF
















#	Bio Design Automation & Bio Manufacturing Automation Companies

bio manufacturing automation:
+ [addgene](https://www.addgene.org/careers/)
+ [Amyris](https://amyris.com/careers/)
+ [Analytikjena](https://www.analytik-jena.com/company/careers/)
+ [Asimov, Inc.](https://www.asimov.io/)
+ [Automata](https://automata.tech/)
	- ***cloud-based robotic laboratory for life sciences, such as biology and biochemistry***
+ BioFab:
	- http://www.uwbiofab.org/
	- http://erc-assoc.org/achievements/biofab-world%E2%80%99s-first-biological-design-build-facility-synthetic-biology
	- Not http://biofabproducts.com/about/
+ [Biolegio B.V.](https://www.biolegio.com/)
+ [BioSero](https://www.biosero.com/about-us/)
+ Boston University:
	- CIDAR Lab
	- Biological Design Center, BDC
+ BP
+ [De Novo DNA](http://www.denovodna.com/)
+ [DSM](https://www.dsm.com/corporate/home.html)
+ [Emerald Cloud Lab](https://www.emeraldcloudlab.com/careers/)
	- ***cloud-based robotic laboratory for life sciences, such as biology and biochemistry***
+ GE Healthcare
+ [genomatica](https://www.genomatica.com/careers/)
+ Genome Foundry
	- https://www.genomefoundry.org/aboutus
	- https://www.concordia.ca/research/genome-foundry.html
+ [Ginkgo Bioworks](https://www.ginkgobioworks.com/about/)
+ [Hamilton Robotics](https://www.hamiltoncompany.com/)
+ [HighRes biosolutions](https://highresbio.com/careers/)
+ [Integrated DNA Technologies](https://www.idtdna.com/pages)
+ [Labcyte](https://www.labcyte.com/)
+ [Lattice Automation](https://www.latticeautomation.com/)
+ Leselagen
+ [lifefoundry](http://www.life-foundry.com/careers.html)
+ Lincoln DNA Foundry
+ [Molecular Devices](https://www.moleculardevices.com/about-us#gref)
+ [New England BioLabs](https://www.neb.com/about-neb/careers)
+ [Novozymes](https://www.novozymes.com/en)
+ [opentrons](https://opentrons.com/jobs)
	- Opentrons Labworks, Inc. 
+ [OriCiro Genomics Inc.](https://www.oriciro.com/)
+ Ozen
+[Ranomics Incorporated](https://www.ranomics.com/careers)
+ [SGIDNA, a Synthetic Genomics, Inc. company](https://www.sgidna.com/careers/)
+ [Strateos, Inc](https://strateos.com/about/)
	- Includes:
		* Transcriptic
		* 3Scan
		* Transcriptic and 3Scan merged to form Strateos.
	- ***cloud-based robotic laboratory for life sciences, such as biology and biochemistry***
+ [Synbicite](http://www.synbicite.com/about-us/)
+ [Synbiobeta](https://synbiobeta.com/job-board/)
+ [Synlogic](https://www.synlogictx.com/our-people/join-us/)
+ [TeselaGen Biotechnology Inc.](https://teselagen.com/)
+ [ThermoFisher Scientific](http://jobs.thermofisher.com/)
+ [Zymergen](https://www.zymergen.com/careers/)








# Biotechnology Companies/Organizations

+ Alector
+ BioMarin
+ CytomX
+ Denali Therapeutics
+ Distributed Bio
+ Droplet Health
+ Encoded Genomics
+ Genetech
+ Haystack Sciences
+ list of biotech and pharmaceutical companies:
	- companies that sponsor, participate in the industrial showcase at, [Synthetic Biology: Engineering, Evolution & Design, SEED](https://synbioconference.org/2022)
+ Nkarta
+ OncoMed
+ Roche Molecular Systems
+ Senti Biosciences
+ Viewpoint Therapeutics











